modules/helm-adding-helm-chart-repositories.adoc

```diff
@@ -43,3 +43,13 @@ EOF
 ----
 +
 . Navigate to  the *Developer Catalog* in the web console to verify that the helm charts from the Azure chart repository are displayed.
++
+Use the *Chart Repositories* filter to search for the required Helm Chart
++
+.Chart Repositories filter
+image::helm-chart-repo-filer.png[]
++
+[NOTE]
+====
+If an admin removes all the chart repos, the user cannot view the ability to add or view Helm Charts from the navigation or forms.
```

Comment:
We will need to make this note more appropriate for the docs, and can not use it as is:

s/an admin/a cluster administrator
s/cannot view the ability to add or view Helm Charts from the navigation or forms./ cannot view the Helm option in the *+Add* view, *Developer Catalog*, and left navigation panel (Please test and validate that this is correct.) and thus cannot add Helm Charts.

Reply:
This is not part of this PR, but can you please make this change too on Line 8:

s/You can add/As a cluster administrator, you can add...

Reply:
Fixed the comment in the new commit.